src/snowflake/snowpark/modin/plugin/_internal/window_utils.py

```diff
@@ -5,18 +5,35 @@
 # This file contains utils functions used by the groupby functionalities.
 #
 #
+from enum import Enum
 from typing import Any
 
 from snowflake.snowpark.modin.plugin.utils.error_message import ErrorMessage
 
-IMPLEMENTED_ROLLING_AGG_FUNCS = ["sum", "mean", "var", "std", "min", "max"]
 
+class WindowFunction(Enum):
+    """
+    Type of window function.
+
+    Attributes:
+        EXPANDING (str): Represents the expanding window.
+        ROLLING (str): Represents the rolling window.
+        WEIGHTED (str): Represents the weighted window.
+        EXPONENTIAL (str): Represents the exponential weighted window.
+    """
+
+    EXPANDING = "expanding"
+    ROLLING = "rolling"
+    WEIGHTED = "weighted"
```

Comment:
i do not see the WEIGHTED and  EXPONENTIAL is used today. is it for some other window that we might support later? if yes, let's add them later to be clear

Reply:
Yes they are for later, I can remove them